examples/computer_vision/cifar10_tf_keras/cifar_model.py

```diff
@@ -6,11 +6,16 @@
 import numpy as np
 import tensorflow as tf
 from data import NUM_CLASSES
+from packaging import version
 from tensorflow.keras.layers import Activation, Conv2D, Dense, Dropout, Flatten, MaxPooling2D
 from tensorflow.keras.losses import categorical_crossentropy
 from tensorflow.keras.metrics import categorical_accuracy
 from tensorflow.keras.models import Sequential
-from tensorflow.keras.optimizers import RMSprop
+
+if version.parse("2.11.0") <= version.parse(tf.__version__):
```

Comment:
nit: It's confusing to me to read a conditional as:
```
if 10 < x:
```
rather than 
```
if x > 10:
```
But I can't find anything in the style guide that disallows it, so to each their own. If this was intentional or an idiom I'm unfamiliar with, then please ignore my comment. If an artifact of the dev process and unintentional, here's a nudge to change it.